dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230420AlterVarcharLengthOfLockedByCol.java

```diff
@@ -0,0 +1,34 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+
+import java.sql.SQLException;
+
+public class Task230420AlterVarcharLengthOfLockedByCol implements StartupTask {
```

Comment:
Doc

Reply:
I'll add it!

Reply:
Done
<img width="741" alt="image" src="https://user-images.githubusercontent.com/127987858/233796531-9fbd018e-e1ef-4e0a-9983-ccc2a3453f25.png">